docker-compose.yaml

```diff
@@ -58,7 +58,7 @@ services:
       - OPENSEARCH_USER=admin
       - OPENSEARCH_PASS=admin
       - OPENSEARCH_INDEX=test_index
-      - MAPPING_DEFINITION='{"id":.sophoraId,"externalid":.externalId,"uuid":.uuid,"type":.type,"title":.title,"description":.["description"],"longDescription":.["longDescription"],"embedText":(.title+""+.longDescription),"availableFrom":.availableFrom,"availableTo":.availableTo,"firstPublicationDate":.firstPublicationDate,"keywords":.keywords,"language":.language,"duration":.durationSeconds,"thematicCategories":.thematicCategories,"genreCategory":.genreCategory,"subgenreCategories":.subgenreCategories,"filterCategory":.filterCategory,"structurePath":.structurePath,"site":.site,"domain":.domain,"teaserimage":.teaserImage.url,"fskAgeRating":.fskAgeRating,"contentRatings":.contentRatings,"geoAvailability":.geoAvailability,"all-MiniLM-L6-v2_title_desc_nokeywords":.["all-MiniLM-L6-v2_title_desc_nokeywords"],"sophoraid":.sophoraId}'
+      - MAPPING_DEFINITION="{\"id\":.externalId,\"externalid\":.externalId,\"uuid\":.uuid,\"sophoraid\":.sophoraId,\"type\":.type,\"title\":.title,\"description\":.description,\"longDescription\":.longDescription,\"embedText\":(.title+\".\n\"+.longDescription),\"availableFrom\":.availableFrom,\"availableTo\":.availableTo,\"firstPublicationDate\":.firstPublicationDate,\"keywords\":.keywords,\"language\":.language,\"duration\":.durationSeconds,\"thematicCategories\":[.thematicCategories[]|.title],\"genreCategory\":.genreCategory,\"subgenreCategories\":[.subgenreCategories[]|.title],\"filterCategory\":[.filterCategory[]|.title],\"structurePath\":.structurePath,\"site\":.site,\"domain\":.domain,\"teaserimage\":.teaserImage.url,\"fskAgeRating\":.fskAgeRating,\"contentRatings\":.contentRatings,\"geoAvailability\":.geoAvailability}"
```

Comment:
Warum genau wird das hier in dem Ticket umgesetzt? Bzw. warum wird hier überhaupt das Mapping in der Docker Compose verändert?